docusaurus/docs/reactnative/common-content/contexts/channel-context/scroll_to_first_unread_threshold.mdx

```diff
@@ -0,0 +1,6 @@
+Minimum number of unread messages a channel should have, to set the initial scroll position to first unread message.
```

Comment:
maybe replace `, ` with ` in order to` so its `...a channel should have in order to set the...`. I think its more clear that way...